index.ejs

```diff
@@ -12,6 +12,7 @@
   <link rel="stylesheet" href="./base.css">
   <link href="https://cdnjs.cloudflare.com/ajax/libs/slim-select/1.27.1/slimselect.min.css" rel="stylesheet">
   </link>
+  <link rel="manifest" href="manifest.json"/>
```

Comment:
should this be in a separate PR?  No big deal, but if this is required by Safe, maybe we can document it here.